subprojects/composite-builds/src/integTest/groovy/org/gradle/integtests/composite/CompositeBuildContinueOnFailureIntegrationTest.groovy

```diff
@@ -159,4 +142,14 @@ class CompositeBuildContinueOnFailureIntegrationTest extends AbstractCompositeBu
         executed ":buildB:jar", ":buildB:checkContinueFlag"
         outputContains("continueOnFailure = true")
     }
+
+    // With failing tasks and parallel output, we cannot assert that the task was executed only once.
+    // This is because our OutputScrapingExecutionResult doesn't handle the output for interleaved tasks
+    // TODO:DAZ Using build operations instead of output scraping would help.
```

Comment:
Perhaps, but this is a general problem with our test fixtures and parallel tasks. [This code](https://github.com/gradle/gradle/blob/efc57711e95760786090bb4bfe4e573f5dfa8224/subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/OutputScrapingExecutionResult.java#L197-L201) is designed to handle the case where we have `:task \n :task UPTODATE` or similar. But it doesn't work when output is interleaved with parallel execution.

I'm tempted to just ignore the 'no duplicates' check here for now.